src/views/items/item.ts

```diff
@@ -11,13 +11,20 @@ class Item extends vscode.TreeItem {
 
   contextValue = 'item';
 
-  constructor ( obj, label: string, command?: vscode.Command, collapsibleState: vscode.TreeItemCollapsibleState = vscode.TreeItemCollapsibleState.Expanded ) {
+  constructor (
+    obj,
+    label: string,
+    command?: vscode.Command,
+    collapsibleState: vscode.TreeItemCollapsibleState = vscode.TreeItemCollapsibleState.Expanded,
+    description?: string,
+  ) {
```

Comment:
Can you keep the coding style consistent?

Reply:
Sure. I only changed it because line was getting longer than others I saw 👍

Reply:
It was good thinking, but I'm just too used to write code that way.

Reply:
Would be great if you could add `eslint` with your config so we can all know how to format it from now on. Not a biggie if you can't tho

Reply:
I don't really have one, I used eslint in the past and it wasn't worth it for me, I just spent more time fighting the linter. I should set up a formatter some day.